crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -98,3 +104,17 @@ const transform = (
 };
 
 export { transform as default };
+
+function makeErrorEmitter(severity: "warning" | "error", ipc: Ipc) {
+  return function (error: Error | string) {
+    const errorObject = error instanceof Error ? error : new Error(error);
```

Comment:
The stack should be omitted if this isn't an error, so we don't confuse our creation site with the throw site.